aws-curl

```diff
@@ -499,6 +507,22 @@ fi
 if [ "$EC2_CREDS" = 1 ]; then
     ec2_import_creds
 fi
+get_cred_value() {
```

Comment:
Suggest to name it as ini_get_value, please assign named local variables for $1 and $2, format it similar to other functions in this file and move up to function.

I guess credentials file can have zero or more spaces around `=` sign and this code works just for one specific scenario. Suggest to adjust syntax to tolerate extraneous spaces around equal sign, also, tolerate if there are no any spaces.